src/takeLast.js

```diff
@@ -0,0 +1,41 @@
+var _curry2 = require('./internal/_curry2');
+var drop = require('./drop');
+
+/**
+ * Returns a new list containing the last `n` elements of the given list.
+ * If `n > list.length`, returns a list of `list.length` elements.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig Number -> [a] -> [a]
+ * @sig Number -> String -> String
+ * @param {Number} n The number of elements to return.
+ * @param {Array} xs The collection to consider.
+ * @return {Array}
+ * @example
+ *
+ *      R.takeLast(1, ['foo', 'bar', 'baz']); //=> ['baz']
+ *      R.takeLast(2, ['foo', 'bar', 'baz']); //=> ['for', 'baz']
+ *      R.takeLast(3, ['foo', 'bar', 'baz']); //=> ['foo', 'bar', 'baz']
+ *      R.takeLast(4, ['foo', 'bar', 'baz']); //=> ['foo', 'bar', 'baz']
+ *      R.takeLast(3, 'ramda');               //=> 'mda'
+ *
+ *      var personnel = [
+ *        'Dave Brubeck',
+ *        'Paul Desmond',
+ *        'Eugene Wright',
+ *        'Joe Morello',
+ *        'Gerry Mulligan',
+ *        'Bob Bates',
+ *        'Joe Dodge',
+ *        'Ron Crotty'
+ *      ];
+ *
+ *      var takeLastFive = R.takeLast(5);
+ *      takeFive(personnel);
```

Comment:
The name here is wrong.  Should be `takeLastFive`.  But there's probably no reason to carry this joke over from `take`.  The whole idea is just a play on the classic Paul Desmond-composed Dave Brubeck quartet piece, [Take Five][tf], one of the best known Jazz pieces of all times.  I would just drop it here.


  [tf]: https://en.wikipedia.org/wiki/Take_Five